content/blog/environment-variables-in-golang/index.md

```diff
@@ -14,28 +14,28 @@ This tutorial assumes you have:
 *   Latest Golang version installed on your system
 *   A few minutes of your time.
 
-In this article, we will see what are the usage of environment variables in an application, and how to access the environment variables in a Go application with different inbuilt and third-party packages.
+In this article, we will know about environment variables and why to use them. And will access them in a Go application using inbuilt and third-party packages.
 
 ## What are environment variables?
 
-Environment variables are key-value pair on a system-wide level, that can be accessed by running processes on the system. Environment variables are often used to make the same program behave differently in separate deploy environments like PROD, DEV, or TEST. 
-Storing configuration in the environment is one of the principles of a twelve-factor app it enables applications to be built with portability.
+Environment variables are key-value pair on a system-wide level, and running processes can access that. These are often used to make the same program behave differently in different deploy environments like PROD, DEV, or TEST. 
+Storing configuration in the environment is one of the principles of a twelve-factor app it enables applications to be portable.
```

Comment:
should be `Storing configuration in the environment is one of the principles of a twelve-factor app. It enables applications to be portable.`